app/controller/sdl/Abstract/Controller.js

```diff
@@ -791,6 +791,8 @@ SDL.SDLController = Em.Object.extend(
       );
       SDL.SDLModel.data.set('interactionData.vrHelpTitle', null);
       SDL.SDLModel.data.set('interactionData.vrHelp', null);
+      clearTimeout(SDL.SDLModel.promptTimeout);
+      SDL.SDLModel.set('timeoutPromprtCallback', undefined);
```

Comment:
```suggestion
      SDL.SDLModel.set('timeoutPromptCallback', undefined);
```

Typo in variable name

Reply:
@iCollin Fixed in 9198805